features/layout/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -89,11 +88,12 @@ export function SidebarNavigation() {
             ))}
           </ul>
           <ul className={styles.list}>
-            <MenuItemButton
+            <MenuItemLink
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              href="mailto:support@prolog-app.com?subject=Support Request: "
+              isActive={false}
```

Comment:
Change from `<MenuItemButton />` to `<MenuItemLink/>` so that we can reference the `href` as a prop. This makes testing easier since we can reference the `href` prop in the test.

There's other solutions such as utilizing `window`, however, that makes testing more complex since we wouldn't want our testing suite handling out of app scenarios. Also, considering we are using next.js, I would stray from utilizing `window` since `window` isn't defined at server check.